toucan_connectors/toucan_connector.py

```diff
@@ -252,9 +256,15 @@ def get_connector_secrets_form(cls) -> Optional[ConnectorSecretsForm]:
     """
     if hasattr(cls, 'get_connector_secrets_form'):
         return getattr(cls, 'get_connector_secrets_form')()
+    return None
+
 
+_UI_HIDDEN: dict[str, Any] = {'ui.hidden': True}
```

Comment:
great to have this.... question, is it not better to have it as a global variable that all connectors should import when they have fields with 'ui.hidden' ?

Reply:
I dont think so, as mypy is only running on this file and on snowflake_connector. Maybe someday if the pattern is repeated in too many places